tests/foreman/ui/test_architecture.py

```diff
@@ -48,11 +53,12 @@ def test_positive_create_arch_with_different_names(self, name):
 
         """
         with Session(self.browser) as session:
-            make_arch(session, name=name)
-            self.assertIsNotNone(self.architecture.search(name))
+            for name in generate_strings_list(remove_str='html'):
```

Comment:
html inputs cannot be searched using the existing search box functionality.  So removing those from the creation tests in all the places